barretenberg/sol/test/honk/ECDSAZK.t.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Aztec
+pragma solidity >=0.8.4;
+
+import {TestBaseHonkZK} from "./TestBaseHonkZK.sol";
+import {EcdsaHonkZKVerifier} from "../../src/honk/instance/EcdsaHonkZK.sol";
+import {DifferentialFuzzer} from "../base/DifferentialFuzzer.sol";
+import {IVerifier} from "../../src/interfaces/IVerifier.sol";
+
+contract EcdsaHonkZKTest is TestBaseHonkZK {
+    function setUp() public override(TestBaseHonkZK) {
+        super.setUp();
+
+        verifier = IVerifier(address(new EcdsaHonkZKVerifier()));
+        fuzzer = fuzzer.with_circuit_type(DifferentialFuzzer.CircuitType.Ecdsa);
+
+        PUBLIC_INPUT_COUNT = 6;
+
+        // Add default inputs to the fuzzer (we will override these in fuzz test)
+        uint256[] memory inputs = new uint256[](6);
+        inputs[0] = uint256(0x67);
+        inputs[1] = uint256(0x6f);
+        inputs[2] = uint256(0x62);
+        inputs[3] = uint256(0x6c);
+        inputs[4] = uint256(0x69);
+        inputs[5] = uint256(0x6e);
+
+        fuzzer = fuzzer.with_inputs(inputs);
+    }
+
+    function testFuzzProof() public {
+        // NOTE we do not fuzz here yet
+        // "goblin"
```

Comment:
I like just "goblin" as comment, I'm going to use that one

Reply:
i'm actually intrigued why this is here (i took the code from the ECSDA.t.sol)

Reply:
goblin is the ascii of the string used in the test !